src/main/java/teammates/common/util/Const.java

```diff
@@ -225,7 +225,9 @@ public static final String getjQueryUiFilePath(boolean isDevServer) {
         public static final String COURSE_DETAILS = "View, edit and send invitation emails to the students in the course";
         public static final String COURSE_EDIT = "Edit Course information and instructor list";
         public static final String COURSE_DELETE = "Delete the course and its corresponding students and sessions";
-        public static final String COURSE_ARCHIVE = "Archive the course so that it will not be shown in the home page any more (you can still access it from the 'Courses' tab)";
+        public static final String COURSE_ARCHIVE =
+                "Archive the course so that it will not be shown in the home page any more "
```

Comment:
Since it is just a string constant, should we start it in the same line as the `=` and break whenever we reach the limit?
If we do this consistently for all string constants, they'll just read like a justified text, it will and also help save brain power on deciding where to put the breaks.